Python Tutorial Exercises(Week 3)/#1.py

```diff
@@ -0,0 +1,19 @@
+'''
+This script takes in an argument n and prints the factorial output, 
+without using any math libraries. A factorial of a number is represented by n!, 
+for example, 3! = 3 x 2 x 1 = 6. 
+This script includes a while loop and a for loop.
+'''
+n = 5
+factorial = 1
+#1 for loop
+for i in range(1, n+1):
+    factorial=factorial*i
+print(factorial)
+
+#1 while loop
+
+while n > 0:
+    factorial = factorial * n
```

Comment:
will this work with this having the same variable name as the for loop variable? Just tried on my end and printed
```
120
14400
```